monitoring/osd-understanding-the-monitoring-stack.adoc

```diff
@@ -0,0 +1,27 @@
+[id="understanding-the-monitoring-stack"]
+= Understanding the monitoring stack
+include::_attributes/attributes-openshift-dedicated.adoc[]
+:context: understanding-the-monitoring-stack
+
+toc::[]
+
+User-workload monitoring lets you monitor your own applications using a different Prometheus instance on your own worker nodes, without interfering with Red Hat Site Reliability Engineer (SRE) platform metrics.
+
+[NOTE]
+Follow the instructions in this section carefully to configure a supported Prometheus instance. Custom Prometheus instances are not supported by {product-title}.
+
+// Understanding the monitoring stack
+include::modules/osd-monitoring-understanding-the-monitoring-stack.adoc[leveloffset=+1]
+include::modules/osd-monitoring-default-monitoring-components.adoc[leveloffset=+2]
+include::modules/osd-monitoring-default-monitoring-targets.adoc[leveloffset=+2]
+include::modules/osd-monitoring-components-for-monitoring-user-defined-projects.adoc[leveloffset=+2]
+include::modules/osd-monitoring-targets-for-user-defined-projects.adoc[leveloffset=+2]
+
+== Additional resources
+
+* link:https://docs.openshift.com/container-platform/4.6/support/remote_health_monitoring/about-remote-health-monitoring.html[About remote health monitoring]
+* xref:../monitoring/osd-accessing-monitoring-for-user-defined-projects.adoc#granting-users-permission-to-monitor-user-defined-projects[Granting users permission to monitor user-defined projects]
```

Comment:
The page that this is linking to is entitled "Accessing monitoring for user-defined projects". You could update this cross-reference to state that.

Reply:
Yes, and I fixed the ID as well. Thanks!